Are we pulling in @types/lodash for the wrong Lodash version?

The web console uses Lodash 3.10.1, which has `_.trunc`.

```
ERROR in ./src/components/order-service/order-service.controller.ts
(111,26): error TS2339: Property 'trunc' does not exist on type 'LoDashStatic'.
```

https://lodash.com/docs/3.10.1#trunc

Is 4.x the correct `@types/lodash` version?

https://github.com/openshift/origin-web-catalog/blob/master/package.json#L52

cc @jeff-phillips-18

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
     "@types/es6-shim": "^0.31.32",
     "@types/jasmine": "^2.5.38",
     "@types/jquery": "^2.0.33",
-    "@types/lodash": "4.14.51",
+    "@types/lodash": "^3.10.1",
     "@types/node": "^0.0.2",
     "angular-mocks": "1.5.11",
     "autoprefixer": "^6.0.3",

